packages/ui-components/src/__tests__/DeploymentSteps.test.ts

```
@@ -1,579 +1,50 @@
-import type { ConfigSource, GuiDeploymentCfg } from '@rainlanguage/orderbook/js_api';
-import type { DeployModalProps, DisclaimerModalProps } from '../lib/types/modal';
+import type { GuiDeploymentCfg } from '@rainlanguage/orderbook/js_api';
 import userEvent from '@testing-library/user-event';
 import { useGui } from '$lib/hooks/useGui';
-import { getAccount } from '@wagmi/core';
+import { useAccount } from '$lib/providers/wallet/useAccount';
+import * as getDeploymentTransactionArgsModule from '../lib/components/deployment/getDeploymentTransactionArgs';
+import type { HandleAddOrderResult } from '../lib/components/deployment/getDeploymentTransactionArgs';
+import { handleDeployment } from '../lib/utils/handleDeployment';
+import type { Hex } from 'viem';
+import type { DeploymentHandlers, handleDeployment } from '../lib/utils/handleDeployment';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'handleDeployment'. Consider to delete it or rename it.

'handleDeployment' is defined here:


(lint/suspicious/noRedeclare)